langkit/core/metric.py

```diff
@@ -108,8 +108,11 @@ class MultiMetric:
     Callable[[], "MetricCreator"],
     Callable[[], List["MetricCreator"]],
     Callable[[], Metric],
+    Callable[[], MultiMetric],
```

Comment:
These shouldn't be required. The `Metric' is already a union of single and multiple. If it works with single then it should work with multiple